src/ImageSharp.Drawing/Paths/InternalPath.cs

```diff
@@ -203,6 +247,92 @@ public bool PointInPolygon(Vector2 point)
             return oddNodes;
         }
 
+        private static bool BoundingBoxesIntersect(Vector2 line1Start, Vector2 line1End, Vector2 line2Start, Vector2 line2End)
```

Comment:
Can we use the XML docs for private methods also please. Some (many) might see it as overkill but we do most of our really complicated stuff in private methods so it's good to add additional documentation and really helps with intellisense internally